src/components/copy-palette/CopyPalette.tsx

```diff
@@ -1,39 +1,43 @@
-import { Button } from "@components/ui/button.tsx";
+import { Button } from "@/components/shared/ui/button";
 import { tailwindGenerator } from "@lib/tailwindGenerator.ts";
 import { clipboard } from "@utils/clipboard.ts";
 import { Copy } from "lucide-react";
 import { toast } from "sonner";
 import {
   Tooltip,
   TooltipContent,
-  TooltipProvider,
-  TooltipTrigger,
-} from "./ui/tooltip";
+  TooltipProvider
+} from "../shared/ui/tooltip";
+import { Palette } from "@/utils/palettes";
 
 interface CopyPaletteProps {
   colors: { color: string; text: string }[];
 }
 
 export function CopyPalette({ colors }: CopyPaletteProps) {
+
+  const handlerClick = async () => {
+    const response = await tailwindGenerator(colors);
+
+    const palette: Palette | string = response[1];
+
+    clipboard(JSON.stringify(palette));
+    toast(`Palette copied correctly! 🐭`);
+  };
+
   return (
     <TooltipProvider>
       <Tooltip delayDuration={200}>
-        <TooltipTrigger>
           <Button
+            aria-label="copy button"
```

Comment:
Could be "Copy palette" for additional contextual information.

Reply:
You are correct; it should be copy palette.